tensorflow/lite/kernels/internal/tensor_utils.cc

```diff
@@ -13,10 +13,13 @@ See the License for the specific language governing permissions and
 limitations under the License.
 ==============================================================================
 
-This is a fork from Tensorflow Lite for TensorflowLite Micro usage
+internal/reference/portable_tensor_utils.h has the implementation of the
```

Comment:
let's keep this comment block separate from the copyright header.

Our style is to have multiline comments also be with `//`.

So something like:
```cc
/* 
copyright header
*/

// The fact that this cc file only has header includes in intentional.
//
// internal/reference/portable_tensor_utils.h has the implementation of the functions declared
// in internal/portable_tensor_utils.h. This somewhat confusing setup is derived from how the
// code is organized in TfLite where it is used to select between NEON, SSE and portable
// implementaitons.
// See https://github.com/tensorflow/tensorflow/blob/d76c23975c4a3a0d7987cfe3f45c76566df06180/tensorflow/lite/kernels/internal/tensor_utils.cc
// for how the code is written in TfLite.

Reply:
Done!